tests/integration/run_tests.py

```diff
@@ -388,8 +388,9 @@ def define_tests():
             f"{run} {simple} --local --vars OUTPUT_PATH=./{OUTPUT_DIR}",
             ReturnCodeCond(),
         ),
-        "merlin-example simple_chain": (
-            f"merlin-example simple_chain ; {run} simple_chain.yaml --local --vars OUTPUT_PATH=./{OUTPUT_DIR} ; rm simple_chain.yaml",
+        "example failure": (f"merlin example failure", RegexCond("not found"),),
+        "example simple_chain": (
+            f"merlin example simple_chain ; {run} simple_chain.yaml --local --vars OUTPUT_PATH=./{OUTPUT_DIR} ; rm simple_chain.yaml",
```

Comment:
does this need to cd into the simple_chain example directory first? or maybe it's run simple_chain/simple_chain.yaml

Reply:
The way I built it (for now) is that all examples make their own directory, except for simple chain because it has no accompanying scripts.